features/step-definitions/search.steps.js

```diff
@@ -31,19 +32,37 @@ module.exports = function() {
           ));
         });
     });
-  // TODO: add steps for valid stock search
-  // TODO: add steps for search with ENTER
-  this.Given(/^focus is on the searchbar$/, () => {
+
+  // Scenario: Search for a valid stock
+  this.When(/^I type text that is a valid stock symbol$/, () => {
     return driver.findElement(homeElements.searchText).then(search => {
-      return search.click();
+      return search.sendKeys(exampleSearchText);
+    });
+  });
+  this.Then(/^I should see some stock results$/, () => {
+    return driver.wait(until.elementLocated(
+      by.css('div.stockDataRegion'), 10000)
```

Comment:
- [ ] Move `by.css('div.stockDataRegion')` and other locators to page-objects for consistency
https://github.com/gregdardis/stock-market-search/blob/dbd190a5aed38eca19a38da1ec56eb099fcd5b6d/features/page-objects/home.js#L6
- [ ] I think you can delete the second argument to `until`, since it is optional and 10s is the default timeout
- [ ] Missing semicolon?

Reply:
Can't find the missing semicolon, other than that I dealt with these